src/index.ts

```diff
@@ -140,12 +140,13 @@ interface Route {
 }
 
 interface Response extends ServerResponse {
-	locals: {[key: string]: any}
-	log: () => this
-	typeLen: (type: string, len: number) => this
-	send: (type: string, body: string | Buffer) => this
-	format: (map: {[key: string]: () => void }) => this
+	locals: { [key: string]: any }
+    log: () => this
+    typeLen: (type: string, len: number) => this
+    send: (type: string, body: string | Buffer) => this
+    format: (map: { [key: string]: () => void }) => this
```

Comment:
Use tabs, not spaces.
```suggestion
	log: () => this
	typeLen: (type: string, len: number) => this
	send: (type: string, body: string | Buffer) => this
	format: (map: { [key: string]: () => void }) => this
```